### PR TITLE
Documentación del asistente que exporta a Excel un documento N57

### PR DESCRIPTION
**Pull request by aerophantom**
### Description:

- New documentation: how to export a N57 file.

### Tasks:

- [x] Add documentation.
- [x] Add images.
- [x] Translations

### Links:

* [Generar fitxer Excel (.xlsx) d'un informe N57 CA](http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/ca/facturacion/cobros/#generar-fitxer-excel-xlsx-dun-informe-n57)
* [Generar fitxer Excel (.xlsx) d'un informe N57 ES](http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/es/facturacion/cobros/#generar-fichero-excel-xlsx-de-un-informe-n57)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/facturacion/cobros.md [hunk 1]
```diff
@@ -200,3 +200,25 @@ Els passos són els següents:
...
+
+![](_static/cobros/menu_wizard_generate_excel_n57_1.png)
+
+Un cop s'obri la vista dels informes, podrem generar l'informe seguint les següents passes:
+
+![](_static/cobros/form_wizard_generate_excel_n57_2.png)
+
+1. Seleccionem l'informe que volem. També es pot obtenir l'Excel des de la vista de formulari ja que les passes 2 i 3 seran les mateixes.
+
+    !!! danger "Perill"
```
Comment (jaumef) [inline docs/facturacion/cobros.md]: No cal que sigui danger, pots posar warining i dir que passa enlloc de dir que es faci servir un sol registre.

i.e.
Quan s'utilitza desde la vista d'arbre, per defecte es seleccionen tots els registres llistats (amb el límit)

File: docs/facturacion/cobros.md [hunk 2]
```diff
@@ -200,3 +200,25 @@ Els passos són els següents:
...
+        Si estem a la vista d'arbre, és molt important que abans de fer la passa dos, assegurar-se de que tenim un registre seleccionat.
+
+2. Obrirem l'assisten clicant el boto `Action`
```
Comment (jaumef) [inline docs/facturacion/cobros.md]: assisten → assistent
clicant → fent click
boto → botó

----

Es prem el botó "`Action`" després de seleccionar el tipus d'informe?

La localització del informe hauria d'anar abans de l'utilització

File: docs/facturacion/cobros.md [hunk 3]
```diff
@@ -200,3 +200,25 @@ Els passos són els següents:
...
+
+3. Generarem l'Excel clicant al botó `Exportar`. Si el volem desar farem clicarem el botó amb l'ícona de desar i si només el volem obrir clicarem el botó `Obrir`.
```
Comment (jaumef) [inline docs/facturacion/cobros.md]: Parlem amb 3a persona en documentació.

Generarem → Es genera

Crec que aquesta frase no cal:
Si el volem desar farem clicarem el botó amb l'ícona de desar i si només el volem obrir clicarem el botó `Obrir`.

File: docs/facturacion/cobros.md [hunk 4]
```diff
@@ -200,3 +200,25 @@ Els passos són els següents:
...
+
+    !!! note "Nota"
+        Si es desa l'arxiu, es farà amb el nom _n57_NOM_FITXER.xlsx_
```
Comment (jaumef) [inline docs/facturacion/cobros.md]: El nom està fixat en l'assistent? Diria que quan guardes es pot canviar.

Pots posar com "info" que **per defecte** es guarda amb aquest nom

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/facturacion/cobros.md [hunk 5]
```diff
@@ -200,3 +200,25 @@ Els passos són els següents:
 
 !!! warning "Atenció"
     Només es pot desfer l'agrupació de factures que no hagin estat pagades.
+
+### Generar fitxer Excel (_.xlsx_) d'un informe N57
```
Comment (guilleJB) [inline docs/facturacion/cobros.md]: Afegir a la part superior `## Cobros Ventanilla Norma57`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docs_wizard_excel_n57/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**